Modules/mailer/Controller/MailerController.php

```diff
@@ -77,11 +77,17 @@ public function sendAction($id_space) {
         $subject = $this->request->getParameter("subject");
         $content = $this->request->getParameter("content");
 
-        $modelConfig = new CoreConfig();
-        $mailerSetCopyToFrom = $modelConfig->getParamSpace("MailerSetCopyToFrom", $id_space);
-        $mailerSetCopyToFromBool = false;
-        if($mailerSetCopyToFrom == 1){
-            $mailerSetCopyToFromBool = true;
+        if (!in_array($to, array("all", "managers"))) {
+            $toEx = explode("_", $to);
```

Comment:
explode by underscore ?  the $to string contains a list of mails with underscore as delimiter ?
weird ? how is it set ? usually, we have comma separated list in this case

Reply:
I kept the mechanism Sylvain was using. In fact, the input select where the user selects to which group he wants to send the email returns values like "a_1" for an area which id is ,1 or "r_2" for a resource which id is 2.
He could also have extracted the 1st char (with a substr()), but not sure it would have been more sensible. explode() also appears to be pretty fast.